public/app/features/alerting/unified/components/extensions/AlertInstanceExtensionPoint.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+
+import { PluginExtensionLink, PluginExtensionPoints } from '@grafana/data';
+import { getPluginLinkExtensions } from '@grafana/runtime';
+import { Dropdown, IconButton } from '@grafana/ui';
+import { ConfirmNavigationModal } from 'app/features/explore/extensions/ConfirmNavigationModal';
+import { Alert, CombinedRule } from 'app/types/unified-alerting';
+
+import { AlertExtensionPointMenu } from './AlertInstanceExtensionPointMenu';
+
+interface AlertInstanceExtensionPointProps {
+  rule?: CombinedRule;
+  instance: Alert;
+  extensionPointId: PluginExtensionPoints;
+}
+
+export const AlertInstanceExtensionPoint = ({
+  rule,
+  instance,
+  extensionPointId,
+}: AlertInstanceExtensionPointProps): ReactElement | null => {
+  const [selectedExtension, setSelectedExtension] = useState<PluginExtensionLink | undefined>();
+  const context = { instance, rule };
+  const extensions = useExtensionLinks(context, extensionPointId);
+
+  const menu = <AlertExtensionPointMenu extensions={extensions} onSelect={setSelectedExtension} />;
+  return (
+    <>
+      <Dropdown placement="bottom-start" overlay={menu}>
+        <IconButton name="ellipsis-v" aria-label="Actions" disabled={extensions.length === 0} variant="secondary" />
```

Comment:
I believe we can improve this part of the UI; there's generally two patterns we tend to use in Alerting.

1. We don't show the dropdown at all if no actions can be taken, a disabled button with no explanation is a tad confusing :)
2. We always show the dropdown, with all of the actions but disable the actions the user has no permissions for are cannot take for some other reason.

If 2 we try to wrap the item with a tooltip explaining why we've disabled the action if we know the reason for it (insufficient permissions, x is not installed / configured, etc)

Reply:
If modifications need to happen to support the second pattern let's go with the first one to unblock this PR – that should be good from a UX perspective :)